contexts/ed25519-signature-2020-v1.json

```
@@ -14,6 +14,10 @@
           "@id": "https://w3id.org/security#controller",
           "@type": "@id"
         },
+        "revoked": {
+          "@id": "https://w3id.org/security#revoked",
+          "@type": "xsd:dateTime"
```

I think this needs to be a full IRI:
```suggestion
          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
```

---

Did testing not catch this because xsd was defined higher up in the test case?

---

Not sure. Good catch tho!